acj/static/modules/assignment/assignment-module_spec.js

```diff
@@ -47,7 +47,12 @@ describe('course-module', function () {
 		"description": null,
 		"id": 1,
 		"modified": "Fri, 09 Jan 2015 17:23:59 -0000",
-		"name": "Test Course"
+		"name": "Test Course",
+        "year": 2015,
```

Comment:
eaf78d522cb895b08bfec89d26c87625742bb219 I converted all tab intention for .html and .js files to 4 spaces to match the indention for the .py files 

I realize the dev doc says that the project uses tabs for indention but none of the python files were following it so I opted to go in favor of spaces (the html and js files were mixed indention)